Build/README.md

```diff
@@ -0,0 +1,99 @@
+# Build scripts
+
+## Setup CircleCI
+
+This is how to make it build inside CircleCI. The licenses are needed to be
+added via environment variables inside the project settings within CircleCI.
+
+### Obtain Unity license .ulf files
+
+Repeat the following steps for the following versions:
+
+- 2018.4.14f1
+- 2019.2.11f1
+- 2020.1.0b6-linux-il2cpp
+
+1. Run `Build/local_docker_run.ps1` in PowerShell with `-UnityVersion` parameter
+    being set to Unity version, ex:
+
+    ```powershell
+    # Assuming working directory is at repo root
+    &'.\Build\local_docker_run.ps1' -UnityVersion '2019.2.11f1'
+    ```
+
+    If running Linux or MacOS, you can install [PowerShell Core](https://github.com/PowerShell/PowerShell)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-content-indent] Don’t use mixed indentation for children, remove 1 space](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5459451)